env_vars.sh

```diff
@@ -1,3 +1,8 @@
 # this is user-specified directory to store GigaSpeech dataset
 export GIGA_SPEECH_RELEASE_URL=oss://speechcolab/GigaSpeech/release/GigaSpeech
 export GIGA_SPEECH_LOCAL_ROOT=/Users/jerry/work/git/GigaSpeech # this path needs to have at least XXX G free space
```

Comment:
Is it possible to change `GIGA_SPEECH_LOCAL_ROOT` to a local directory, e.g., relative to home directory so that it will run out of the box? @dophist